momento-sdk/src/main/java/momento/sdk/messages/CacheSetIfNotExistsResponse.java

```diff
@@ -0,0 +1,109 @@
+package momento.sdk.messages;
+
+import com.google.protobuf.ByteString;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import momento.sdk.exceptions.SdkException;
+import momento.sdk.internal.StringHelpers;
+
+/**
+ * Parent response type for a cache setIfNotExists request. The response object is resolved to a
+ * type-safe object of one of the following subtypes:
+ *
+ * <p>- {Stored} - {NotStored} - {Error}
+ *
+ * <p>`instanceOf` type guards are used to operate on the appropriate subtype @Example For example:
+ *
+ * <p>``` if (response.isInstanceOf(CacheSetIfNotExists.Error) { // Handle error as appropriate. The
```

Comment:
.isInstanceOf() is specifically an assertj method. You'll need `if (response instanceOf CacheSetIfNotExists.Error)`

Reply:
whoa how was that even available on the main classpath?

Reply:
It was just in a comment. I assumed the code fragment was pulled from a test, but I could be wrong.

Reply:
roger